docs/examples/vector_stores/QdrantIndexDemo.ipynb

```diff
@@ -32,7 +32,7 @@
    "metadata": {},
```

Comment:
<p>could you keep the output</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/10786/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>